cmd/sbserver/main.go

```diff
@@ -220,9 +220,10 @@ const (
 )
 
 var (
-	apiKeyFlag   = flag.String("apikey", "", "specify your Safe Browsing API key")
-	srvAddrFlag  = flag.String("srvaddr", "localhost:8080", "TCP network address the HTTP server should use")
-	databaseFlag = flag.String("db", "", "path to the Safe Browsing database.")
+	apiKeyFlag       = flag.String("apikey", "", "specify your Safe Browsing API key")
+	srvAddrFlag      = flag.String("srvaddr", "localhost:8080", "TCP network address the HTTP server should use")
+	databaseFlag     = flag.String("db", "", "path to the Safe Browsing database.")
+	updatePeriodFlag = flag.Duration("updateperiod", safebrowsing.DefaultUpdatePeriod, "the update period for the Safe Browsing database in minutes.")
```

Comment:
Remove " in minutes". The flag.Duration requires a time unit to be passed, so the following are all valid: "10s", "20m", "1h"